src/App.scss

```diff
@@ -5,4 +5,7 @@
     margin-top: 64px;
     display: flex;
   }
+  .ant-list-item-meta-title > a {
+    color: #1890ff;
```

Comment:
what about the ``hover`` state? :)

Reply:
It's set to the same color by default. Do you think I should change it to a different color?

Reply:
the global link hover color is different though, we'd like to have the same style for all the links

Reply:
Aah. okay. I'll fix that

Reply:
I've updated the pr.